.travis.yml

```diff
@@ -0,0 +1,19 @@
+if: branch = master and type IN (pull_request)
+language: cpp
+services:
+  - docker
+env:
+before_install: true
+install: true
+before_script: true
+script: true
+jobs:
+  include:
+    - stage: optional
+      script: ci/rpm7/build.sh -o -p $(pwd)
```

Comment:
Looking to https://travis-ci.org/telefonicaid/fiware-orion/jobs/388748666 I see:

![imagen](https://user-images.githubusercontent.com/1534240/41059421-d11acaba-69cc-11e8-8178-91efb63f5623.png)

So it seems there is some problem passing retval from script... This stage should be marked as failed in travis (this is normal, as the style problem that causes will not be solved until https://github.com/telefonicaid/fiware-orion/pull/3200 get merged).

Reply:
Not a problem. It was done especially because Travis will fail all build if one stage exit with 1. So when pr will be landed, will turn exit code to 1.

Reply:
In addition, looking to
https://travis-ci.org/telefonicaid/fiware-orion/builds/388748664

![imagen](https://user-images.githubusercontent.com/1534240/41059612-612d3e8a-69cd-11e8-8352-7ea0bf9be1fa.png)

Optiona, Unit and Functional corresponds to stages in .travis.yml. However, what about Test? I somekind of leftover?

Reply:
Stages itself is a BETA feature, so in this case, Test is a legacy step. In the normal way, Travis adopted to use one stage  and matrix environment variables, like described here: https://docs.travis-ci.com/user/customizing-the-build/#Build-Matrix

Reply:
NTC